DTS-STN/Service-Canada-Labs
# Update ContextualAlert on homepage

# [125823 - Move Alert](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=125823)

- Removed SCLabs Alert component and replaced with SCDS ContextualAlert component
- Repositioned ContextualAlert to be above Card section

## Test Instructions

1. See that Alert positioning and styling match design found [here](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?type=design&node-id=11385-29412&t=bE5yaITGZMkzJIwr-0)

## Definition of Done
